Library/Formula/cdrtools.rb

```diff
@@ -4,7 +4,10 @@ class Cdrtools < Formula
 
   stable do
     url "https://downloads.sourceforge.net/project/cdrtools/cdrtools-3.01.tar.bz2"
+    mirror "https://www.mirrorservice.org/sites/downloads.sourceforge.net/c/cd/cdrtools/cdrtools-3.01.tar.bz2"
+    mirror "https://fossies.org/linux/misc/cdrtools-3.01.tar.bz2"
     sha256 "ed282eb6276c4154ce6a0b5dee0bdb81940d0cbbfc7d03f769c4735ef5f5860f"
+    patch :p0, :DATA
```

Comment:
Sorry, I don't understand what `3.02a02` is, is it a stable version of this software? We just need to have patches submitted upstream where there are bugs upstream or a comment explaining in detail why that isn't necessary. Thanks!